manual/root_files/index.md

```diff
@@ -100,19 +100,41 @@ def closeAtDestruct(hist):
 {% endhighlight %}
 
 
-### Displaying the content of a ROOT file
+### Opening and inspecting a ROOT file
 
-Apart from [`rootls`]({{ '/manual/root_files/#root-command-line-tools' | relative_url }} ) and the [object browser]( {{ '/manual/root_files/#root-object-browser' | relative_url }} ) introduced below, `TFile::ls()` lists what is in the ROOT file.
+Use [TFile::Open()](https://root.cern/doc/master/classTFile.html#ad8870806a04da2c2f4aa02bee4ec6833){:target="_blank"} to open a ROOT file.
+While this operation might return a valid pointer to a `TFile` object,
+this object might not be able to access data, for instance because ROOT was unable to open the file in the filesystem.
+Use [TObject::IsZombie()](https://root.cern/doc/master/classTObject.html#aaa8418b9b6692a12d8d0e500c51911bf){:target="_blank"} to check whether the ROOT file was successfully opened.
 
 {% highlight C++ %}
-root [0] std::unique_ptr<TFile> myFile( TFile::Open("file.root", "RECREATE") );
-root [1] myFile->ls()
+std::unique_ptr<TFile> file( TFile::Open("file.root") );
+if (file->IsZombie()) {
+   std::cerr << "Error opening file" << endl;
+   exit(-1);
+}
+{% endhighlight %}
+
+Apart from [`rootls`]({{ '/manual/root_files/#root-command-line-tools' | relative_url }} ) and the [object browser]( {{ '/manual/root_files/#root-object-browser' | relative_url }} ) introduced below, `TFile::ls()` lists what is in the ROOT file.
 
-TFile**    file.root
- TFile*    file.root
-  KEY: TH1F     MyHist;1 This is a histogram
+{% highlight C++ %}
+root [] std::unique_ptr<TFile> myFile( TFile::Open("file.root") );
+...
+root [] myFile->ls()
+TFile**		file.root	
+ TFile*		file.root	
+  OBJ: TH1F	myHist	histo : 0 at: 0x5592d774c2e0
+  KEY: TH1F	myHist;2	histo [current cycle]
+  KEY: TH1F	myHist;1	histo [backup cycle]
 {% endhighlight %}
 
+By default, existing objects are not replaced when writing new objects with the same name.
+Instead, a new _namecycle_ is created, denoted by `;2`, `;3`, etc.
+When retrieving the object from the file, ROOT will automatically pick the highest namecycle.
+
+Some objects, such as histograms, automatically register themselves with the current TDirectory (e.g. the last TFile opened): these objects will appear as `OBJ` entries, without a namecycle.
```

Comment:
Is there a longer discussion on the ownership model that we can point to from here?

Reply:
Done in https://github.com/root-project/web/pull/735